src/simple_commander/game/invader.py

```diff
@@ -0,0 +1,44 @@
+"""This module describes Invader's behaviour. """
+
+import logging
+from random import randint
+
+from simple_commander.game.unit import Unit
+from simple_commander.utils.constants import DEFAULT_SPEED, STEP_INTERVAL, UNITS
+
+
+class Invader(Unit):
+
+    def __init__(self, x, y, angle, hits=10, speed=DEFAULT_SPEED, obj_type='',
+                 bullet_type=UNITS.get('bullet_invader', {}).get('type', ''), dimension=0, controller=None):
+        if not obj_type and len(UNITS.get('invader', [])):
+            random_number = randint(0, len(UNITS.get('invader', [])) - 1)
+            obj_type = UNITS.get('invader', [])[random_number].get('type', '')
+            dimension = UNITS.get('invader', [])[random_number].get('dimension', '')
+        super(Invader, self).__init__(x, y, angle, hits, speed, obj_type, bullet_type, dimension, controller=controller)
+
+    def reset(self):
+        self.angle = randint(0, 360)
+        self.compute_new_coordinate(STEP_INTERVAL)
```

Comment:
Please note in TODOs that compute_new_coordinate must take angle into account. So that newly generate unit was rotated to the game field, not borders when it's located close to them